rest/rest_suite_test.go

```diff
@@ -0,0 +1,36 @@
+package rest_test
+
+import (
+	_ "crypto/sha512"
+
+	"github.com/ably/ably-go/rest"
+	. "github.com/ably/ably-go/test/support"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"testing"
+)
+
+func TestRest(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Rest Suite")
+}
+
+var (
+	client  *rest.Client
+	channel *rest.Channel
+)
+
+var _ = BeforeSuite(func() {
+	_, err := TestAppInstance.Create()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
i think it would be more idiomatic to assign a variable rather than using a global from the other package, for example:

```go
import (
	"github.com/ably/ably-go/test/support"
)

var testApp *support.TestApp

var _ = BeforeSuite(func() {
	var err error
	testApp, err = support.CreateTestApp()
	Expect(err).NotTo(HaveOccurred())
})
```

This also avoids cluttering the namespace with globals from the support package.